net-dialup/accel-ppp/accel-ppp-1.12.0_p20210430.ebuild

```diff
@@ -23,7 +23,8 @@ RDEPEND="!libtomcrypt? ( dev-libs/openssl:0= )
 	snmp? ( net-analyzer/net-snmp )
 	dev-libs/libpcre"
 DEPEND="${RDEPEND}
-	valgrind? ( dev-util/valgrind )"
+	valgrind? ( dev-util/valgrind )
+	sys-libs/libucontext"
```

Comment:
libucontext needs to be in RDEPEND too, but this should be elibc_musl? ( ... )

Reply:
Added